### PR TITLE
chore: bump dependencies versions

### PR DESCRIPTION
**Pull request by guillaumelamirand**
**Description**

   - fix integrations tests according to latest sdk

BREAKING CHANGE: use apim version 4

**Additional context**

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `3.0.0-bump-apim-versions-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-keyless/3.0.0-bump-apim-versions-SNAPSHOT/gravitee-policy-keyless-3.0.0-bump-apim-versions-SNAPSHOT.zip)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version: 2.1
 setup: true
 
 orbs:
-    gravitee: gravitee-io/gravitee@2.1
+    gravitee: gravitee-io/gravitee@4.0.0
```
Comment (benoitgravitee) [inline .circleci/config.yml:7]: I think the 4.1.0 is to be used for plugins now @gaetanmaisse do you concur

Comment (gaetanmaisse) [reply .circleci/config.yml:7]: Yes!

**Review by jhaeyaert — APPROVED** [review]

**Pull request merged**